DualView/src/main/java/com/microsoft/device/display/samples/dualview/view/RestaurantAdapter.kt

```diff
@@ -31,13 +31,20 @@ class RestaurantAdapter(
     private val layoutInflater = LayoutInflater.from(context)
 
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int) =
-        RestaurantViewHolder(layoutInflater.inflate(R.layout.dual_view_restaurant_item, parent, false))
+        RestaurantViewHolder(
+            layoutInflater.inflate(
+                R.layout.dual_view_restaurant_item,
+                parent,
+                false
+            )
+        )
 
     override fun onBindViewHolder(holder: RestaurantViewHolder, position: Int) {
         holder.bind(
             viewModel.getItem(position)!!,
             onClickRestaurantItem,
-            position == viewModel.selectedPosition.value
+            position == viewModel.selectedPosition.value,
+            position == itemCount - 12
```

Comment:
It would be useful to extract this number into a constant, so we know what it represents

Reply:
Removed because is no longer needed.